src/locales/en.json

```diff
@@ -907,6 +911,7 @@
                 "MultiColor": "This looks like a Multi Material Single Extruder print using {numTools} tools. You should probably review tool mapping",
                 "SingleColor": "This looks like a single color print but you don't have the bypass selected on your MMU. You may want to check the chosen tool"
             },
+            "Tangle": "TANGLE",
```

Comment:
pls make it uppercase only in the fontend/output per css. you cannot be sure, that all translators also type it uppercase in the files.